vagrant/README.md

```diff
@@ -162,6 +162,8 @@ destinations:
 EOF
 ```
 
+And now if we go back to our [marketplace](http://192.168.33.20:18080), everything will work since we allow all services to send traffic to one another.
```

Comment:
@yskopets after enabling mTLS and traffic permissions for all, the new setup with `node.vm.network "forwarded_port"` doesn't work still. any idea why?

Reply:
I see. You're right.

Apparently, `Vagrant`'s  `port-forward` doesn't work the way we need (forwarding to a loopback interface of a guest).

Let's solve it differently then, using regular SSH:
* let's remove `node.vm.network "forwarded_port"` from `Vagrantfile` (since it doesn't work the way we need)
* let's ask users to use the following command to setup port-forwarding:
  ```shell
  vagrant ssh frontend -- -L 127.0.0.1:8080:127.0.0.1:8080
  ```
  this command will setup a listener `127.0.0.1:8080` on the `host` machine which will forward traffic to `127.0.0.1:8080` listener inside the `guest` machine.
  When a user opens `http://127.0.0.1:8080` in a browser, all requests will go directly into `frontend` application (rather than dataplane in front of it), which will allow to do the demo with mTLS enabled.